.github/workflows/upload-e2e-snapshots.yaml

```diff
@@ -0,0 +1,110 @@
+# Copyright 2022 PingCAP, Inc. Licensed under Apache-2.0.
+name: Upload E2E Snapshots
+
+on:
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: "The branch, tag or SHA to create snapshots"
+        required: true
+
+jobs:
+  e2e_test_snapshots:
+    name: Take E2E Test Snapshots
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    strategy:
+      matrix:
+        # test latest features and compatibility of lower version
+        include:
+          - feature_version: 6.0.0
+            tidb_version: nightly
+          - feature_version: 5.0.0
+            tidb_version: v5.0.0
```

Comment:
Are you expecting that the E2E artifacts are different in different version matrixes?

Reply:
No. But there's different test cases in different version matrixes. May have different use for snapshot.